client/src/js/hmm/components/Installer.js

```diff
@@ -17,14 +17,23 @@ const steps = [
 
 class HMMInstall extends React.Component {
 
-    handleRefresh = () => {
-        this.props.onRefresh();
+    getProgress = () => {
+        const progress = 20 * (steps.indexOf(this.props.process.step) + this.props.process.progress);
+        return progress;
+    }
+
+    componentDidUpdate () {
+        const progress = this.getProgress();
+
+        if (progress === 100) {
```

Comment:
To be safe, you should use `prevProps` and the conditional  `(prevProps.progress !== 100 && props.progress === 100)`.

Doing this would probably also require making `getProgress` a pure function that takes `props` as an input so that you can pass `prevProps` to it.

Also, you have an unnecessary definition of `progress` here.